content/_index.md

```diff
@@ -37,7 +37,7 @@ container: "container-fluid"
 <!-- start: Founding Members -->
 <div id="members" class="row featured-section featured-section-members">
   <div class="container">
-  <h2>Founding Members</h2>
+  <h2 class="section-title">Founding Members</h2>
```

Comment:
Do we need to introduce a new class here? If possible, I would prefer when we keep the html simple.